scripts/add_existing_baseyear.py

```diff
@@ -142,6 +142,9 @@ def add_power_capacities_installed_before_baseyear(n, grouping_years, costs, bas
     # Assume that all oil power plants are not CHPs.
     if "H" in snakemake.wildcards.sector_opts.split("-"):
         df_agg = df_agg.query("Set != 'CHP'")
+    else:
+        if "I" not in snakemake.wildcards.sector_opts.split("-"):
+            df_agg.query("Industry" == False, inplace=True)
```

Comment:
```suggestion
    elif "I" not in snakemake.wildcards.sector_opts.split("-") and "Industry" in df_agg.columns:
         df_agg.query("not Industry", inplace=True)
```
very minor :)

But the code should check for existence of "Industry" column, as this cannot be assumed outside the ariadne context.